app/views/_swagger.njk

```diff
@@ -0,0 +1,49 @@
+{% extends "govuk/template.njk" %}
```

Comment:
I think this file should be excluded from the codebase as it gets generated from the template